src/NzbDrone.Core/Applications/Radarr/Radarr.cs

```diff
@@ -174,7 +173,7 @@ private RadarrIndexer BuildRadarrIndexer(IndexerDefinition indexer, DownloadProt
         {
             var cacheKey = $"{Settings.BaseUrl}";
             var schemas = _schemaCache.Get(cacheKey, () => _radarrV3Proxy.GetIndexerSchema(Settings), TimeSpan.FromDays(7));
-            var syncFields = new[] { "baseUrl", "apiPath", "apiKey", "categories", "minimumSeeders", "seedCriteria.seedRatio", "seedCriteria.seedTime" };
+            var syncFields = new[] { "baseUrl", "apiPath", "apiKey", "categories", "multiLanguages", "removeYear", "requiredFlags", "additionalParameters", "minimumSeeders", "seedCriteria.seedRatio", "seedCriteria.seedTime" };
```

Comment:
We shouldn’t add things to these that Prowlarr doesn’t control. I believe it’ll cause Prowlarr to override user settings set in arr

Reply:
knew there was a reason those were removed.
would it be better if they were explicitly listed e.g. `noSyncFields` (with default noSync if undefined field in schema)?
perhaps that can then translate to some fancy UI info text to the users for what fields Full Sync evaluates

Reply:
I fully support dropping older versions lol.

radarr v3 even is mentioned continually on discord as unsupported by staff